viewing_party/party.py

```diff
@@ -1,23 +1,160 @@
 # ------------- WAVE 1 --------------------
-
+import copy
+#Creates a dictionary representing a movie with title, genre and rating.
 def create_movie(title, genre, rating):
-    pass
-
+    if title and genre and rating:
+        return {"title": title, "genre": genre, "rating": rating} #it's a movie_dic
+    return None
 # -----------------------------------------
-# ------------- WAVE 2 --------------------
+#Adds a movie to the user's watched list.
+def add_to_watched(user_data, movie): 
+    if "watched" not in user_data:
+        user_data["watched"] = []
+    user_data["watched"].append(movie)

```

Comment:
The requirements say that the dictionary will have a key "watched" so we can remove the conditional checks here to simplify this function:
```suggestion
    user_data["watched"].append(movie)

```